CONTRIBUTING.md

```diff
@@ -31,12 +31,53 @@ Any module must follow this basic format:
 
 ```js
 module.exports = function ModuleName(options,UI) {
+
   options = options || {};
   options.title = "Title of the Module";
   UI.onSetup(options.step);
   var output;
 
   function draw(input,callback) {
+
+    UI.onDraw(options.step);
+
+    var output = /*do something with the input*/ ;
+
+    this.output = output;
```

Comment:
oy, i still want to be able to do all this automatically rather than needing it in every module. But we're addressing this in #122 -- just want to note it here!